tests/deepdisc/utils/test_parse_arguments.py

```diff
@@ -50,3 +55,13 @@ def test_make_training_arg_parser():
     assert args.scheme == 1
     assert args.stretch == 0.5
     assert args.tl == 1
+
+
+def test_dtype_from_args():
+    assert dtype_from_args(8) is np.uint8
+    assert dtype_from_args(16) is np.int16
+
+    with pytest.raises(ValueError):
+        _ = dtype_from_args(4)
+    with pytest.raises(ValueError):
+        _ = dtype_from_args(32)  # Do we want to support this too?
```

Comment:
Or any kind of floating points?

Reply:
We should support float32 data.  Currently 'raw' data scaling will return a float32 array without regarding the dtype argument.

Reply:
What is the best way to do that? I could map an input of 32 to that type. But will that cause possible confusion with int32s? Alternatively I could have float32 be a default and not raise an error if no type is given.

Any thoughts?

Reply:
or take in a string like "int32"/"float32"?

Reply:
I think defaulting to float32 is ok, it's likely we won't use int32

Reply:
I made the default float32 and updated the tests.